backend/grants/admin.py

```diff
@@ -374,6 +401,41 @@ def user_display_name(self, obj):
             return self.get_user_display_name(obj.user_id)
         return obj.email
 
+    @admin.display(
+        description="C",
+    )
+    def country(self, obj):
+        if obj.traveling_from:
+            country = countries.get(code=obj.traveling_from)
+            if country:
+                return country.emoji
+
+        return ""
+
+    @admin.display(
+        description="S",
+    )
+    def is_speaker(self, obj):
+        if obj.user_id in self.speaker_ids:
+            return "🗣️"
+        return ""
+
+    def get_queryset(self, request):
+
+        qs = super().get_queryset(request)
```

Comment:
nitpick: I imagine another way to do it would be to use `.annotate(is_speaker=Subquery`) but I am not sure it would be more efficient or not than merging in python